packages/pwa-kit-create-app/assets/bootstrap/js/overrides/app/ssr.js

```diff
@@ -38,23 +37,23 @@ const {handler} = runtime.createHandler(options, (app) => {
     // Set HTTP security headers
     app.use(
         helmet({
+            // pwa-kit-runtime ensures that the Content-Security-Policy header always contains the
+            // directives required for PWA Kit to function. Add custom directives here.
             contentSecurityPolicy: {
-                useDefaults: true,
                 directives: {
-                    'img-src': ["'self'", '*.commercecloud.salesforce.com', 'data:'],
-                    'script-src': ["'self'", "'unsafe-eval'", 'storage.googleapis.com'],
-                    'connect-src': ["'self'", 'api.cquotient.com'],
-
-                    // Do not upgrade insecure requests for local development
-                    'upgrade-insecure-requests': isRemote() ? [] : null
+                    scriptSrc: [
+                        'storage.googleapis.com' // used by the service worker in /worker/main.js
+                    ]
                 }
-            },
-            hsts: isRemote()
+            }
         })
     )
 
     // Handle the redirect from SLAS as to avoid error
     app.get('/callback?*', (req, res) => {
+        // This endpoint does nothing and is not expected to change
+        // Thus we cache it for a year to maximize performance
+        res.set('Cache-Control', `max-age=31536000`)
```

Comment:
Unrelated change from the base app copied over. Do we need a mechanism for ensuring this file and the base file from the retail react app stay the same?

Reply:
Yes, I'd say so. Otherwise we'd get weird inconsistencies between the two like the imports vs require differences you've pointed out

Reply:
Nice catch. We most likely introduced this issue with V3, probably in an effort to make our overrides look less complex.